src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -323,7 +323,8 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     <!-- Example, C:\MyProjects\MyProject\bin\Debug\MyAssembly.dll -->
     <TargetPath Condition=" '$(TargetPath)' == '' ">$(TargetDir)$(TargetFileName)</TargetPath>
 
-    <TargetRefPath Condition=" '$(TargetRefPath)' == '' and '$(ProduceReferenceAssembly)' == 'true' ">$([MSBuild]::NormalizePath($(TargetDir), 'ref', $(TargetFileName)))</TargetRefPath>
+    <TargetRefPath Condition=" '$(TargetRefPath)' == '' and '$(ProduceReferenceAssembly)' == 'true' and ('$(ProduceReferenceAssemblyInBin)' == 'true' or '$([MSBuild]::AreFeaturesEnabled(17.0))' != 'true' ) ">$([MSBuild]::NormalizePath($(TargetDir), 'ref', $(TargetFileName)))</TargetRefPath>
```

Comment:
Seems redundant to have this in a change wave _and_ have an opt in for it. By design shouldn't it be one or the other?

Reply:
I expect this to be controversial enough to have a permanent flag. The changewave is just there since we have a big lever for "do stuff the old way".

Reply:
Why not `ProduceReferenceAssemblyInOutDir` or `ProduceReferenceAssemblyInOutputPath`?